server/reports/item-usage/2_5_0/convert_data_js/src/utils.js

```diff
@@ -0,0 +1,70 @@
+import { cleanUpNodes, sortNodes } from "../../../../utils";
+
+const processItemLines = (data, sort, dir) => {
+  data.items.nodes.forEach((item) => {
+    // don't add default values if empty object added
+    if (Object.keys(item).length == 0) {
+      return;
+    }
+    item.monthConsumption = calculateQuantity(
+      data.thisMonthConsumption,
+      item.id
+    );
+    item.lastMonthConsumption = calculateQuantity(
+      data.lastMonthConsumption,
+      item.id
+    );
+    item.twoMonthsAgoConsumption = calculateQuantity(
+      data.twoMonthsAgoConsumption,
+      item.id
+    );
+    item.expiringInSixMonths = calculateQuantity(
+      data.expiringInSixMonths,
+      item.id
+    );
+    item.expiringInTwelveMonths = calculateQuantity(
+      data.expiringInTwelveMonths,
+      item.id
+    );
+    item.stockOnOrder = calculateQuantity(data.stockOnOrder, item.id);
+    item.AMC12 = calculateQuantity(data.AMCTwelve, item.id);
+    item.AMC24 = calculateQuantity(data.AMCTwentyFour, item.id);
+    item.SOH = calculateStatValue(item?.stats?.stockOnHand);
+    item.MOS = calculateStatValue(item?.stats?.availableMonthsOfStockOnHand);
+    item.AMCDefault = calculateStatValue(item?.stats?.averateMonthlyConsumption)
```

Comment:
Can you explain why this is called AMCDefault? Is item stats AMC always using the default lookback period or something? I wonder if there is a clearer name...

Reply:
Actually, what is the point of this column? I don't see it in the OG report or mentioned in the issue?

Reply:
There was confusion about how 'Months Cover' column was calculated.
It is calculated using the consumption over the store preferences lookback period (or the default 3 months lookback period).

Confusion could occur because the months cover might differ from what users would expect it to be given the AMC12 and AMC24 provided. Adding it in shows the users the AMC which is used to calculated the months cover column.

@andreievg mentioned this to me in person, but I hadn't updated the issue. What do you reckon?

Reply:
> Can you explain why this is called AMCDefault? Is item stats AMC always using the default lookback period or something? I wonder if there is a clearer name...

Good point - I'm thinking just 'AMC'

Reply:
Ah ok that makes sense - it should probably show something other than 0 then? 😁 

For testing... this should match the item detail view AMC value right?

![Screenshot 2025-01-21 at 4 09 45 PM](https://github.com/user-attachments/assets/2672f1e2-7956-4f93-9e4a-456ab67cc7fc)

Reply:
> Ah ok that makes sense - it should probably show something other than 0 then? 😁

Yikes! Yes it should be.
Unless of course you haven't used any of those medicines at that store in the last (lookback period) months?
Default is 3 if not set in store preferences.
 
> For testing... this should match the item detail view AMC value right?

Correct - both pull directly from itemStats node